hellomama_registration/utils.py

```diff
@@ -301,15 +302,9 @@ def get_messageset_schedule_sequence(short_name, weeks):
 
 
 def post_message(payload):
-    result = session.post(
-        url="%s/outbound/" % settings.MESSAGE_SENDER_URL,
-        data=json.dumps(payload),
-        headers={
-            'Content-Type': 'application/json',
-            'Authorization': 'Token %s' % (
-                settings.MESSAGE_SENDER_TOKEN,)
-        }
-    ).json()
+    ms_client = MessageSenderApiClient(settings.MESSAGE_SENDER_TOKEN,
```

Comment:
That's exciting, passing in the session causes 15 tests to fail. Will investigate why!